### PR TITLE
more real imports

### PR DESCRIPTION
**Pull request by turadg**
refs: #9149

## Description

Replace more of the `@typedef` "import" with actual `@import`. Should help with docs generation.

### CODE REVIEW

File: packages/agoric-cli/src/commands/inter.js [hunk 1]
```diff
@@ -33,16 +33,16 @@ const bidInvitationShape = harden({
   callPipe: [['makeBidInvitation', M.any()]],
 });
 
-/** @typedef {import('@agoric/vats/tools/board-utils.js').VBankAssetDetail } AssetDescriptor */
-/** @typedef {import('@agoric/smart-wallet/src/smartWallet.js').TryExitOfferAction } TryExitOfferAction */
+/** @import {VBankAssetDetail} from '@agoric/vats/tools/board-utils.js'; */
```
Comment (dckc) [inline packages/agoric-cli/src/commands/inter.js:36]: that's an odd place to define `VBankAssetDetail`. hm.

File: packages/inter-protocol/src/proposals/econ-behaviors.js [hunk 2]
```diff
@@ -67,12 +69,12 @@ export const SECONDS_PER_WEEK = 7n * SECONDS_PER_DAY;
  */
 
 /**
- * @typedef {import('@agoric/zoe/src/zoeService/utils.js').StartedInstanceKit<
+ * @typedef {StartedInstanceKit<
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/econ-behaviors.js:72]: I think only the `@typedef` redefinitions affect generated docs.

I guess this is still cleaner.

**Review by dckc — APPROVED** [review]
yes, good stuff for generated docs

I presume this doesn't complicate cherry-picking for releases unduly. (cc @mhofman @gibson042 )

**Comment by mhofman** [comment]
> I presume this doesn't complicate cherry-picking for releases unduly. (cc @mhofman @gibson042 )

Yeah, wondering if we could split the changes in different commits per different packages (or group of packages) affected.
That would allow partial cherry-picking where appropriate.

**Comment by turadg** [comment]
>Yeah, wondering if we could split the changes in different commits per different packages (or group of packages) affected.

By "we" I think you mean me : ) And I'm open to doing it, if it's worthwhile. 

> That would allow partial cherry-picking where appropriate.

Why is partial necessary? Do the release branches support `@import` or not? (if not then this is moot because none of this will transfer. )

**Comment by mhofman** [comment]
> Why is partial necessary? Do the release branches support `@import` or not? (if not then this is moot because none of this will transfer. )

Good point, we would likely not pick up tsc changes. Please forget about my half thought through request. This will remain another difference between release and master

**Pull request merged**